### ISSUE TITLE
Phénomènes physiques indéterministes

### ISSUE THREAD

**Issue by Ninroot** [Ninroot/reflexio#42]
il y a beaucoup de phénomènes physiques qui sont indéterministes, dès qu'on applique la théorie du chaos, ou qu'on est  à des échelles quantiques (principe d'indétermination de heisenberg).

**Comment by Ninroot** [comment]
Oui, c'est vrai que je ne prends pas du tout en compte ce point pour le moment. Les sujets que j'aborde sont, à ce que je sache, suffisamment macroscopiques pour que leurs fonctionnements puissent être considérés comme déterministes. Peut-être un sujet que j'aborderai dans le futur.

**Comment by floodz26** [comment]
Ben si tu aborde la cognition et la pensée tu n'est plus dans une échelle macroscopique ni déterministe
